lib/attack.go

```diff
@@ -237,7 +243,7 @@ func (a *Attacker) hit(tr Targeter, tm time.Time) *Result {
 		return &res
 	}
 
-	req, err := tgt.Request()
+	req, err := tgt.Request(a.gzip)
```

Comment:
Is there an advantage of performing the compression on every request? If not, I'd suggest to do it once and store the result in the `Target.Body`.